slip-0077.md

```diff
@@ -0,0 +1,62 @@
+# SLIP-0077 : Deterministic blinding key derivation for Confidential Transactions
+
+```
+Number:  SLIP-0077
+Title:   Deterministic blinding key derivation for Confidential Transactions
+Type:    Standard
+Status:  Draft
+Authors: Roman Zeyde <mail@romanzey.de>
+Created: 2019-06-15
+```
+
+## Abstract
+
+This document describes a method for blinding key derivation
+for Confidential Transactions, using a determinstic hierarchy.
+
+## General design
+
+In confidential transactions, the sender and the receiver use ECDH to derive a shared nonce, which is then used for hiding/recovering of the actual value and asset type being transacted.
+In Elements/Liquid, the receiver uses the following derivation scheme for his ECDH public/private keys:
+
+```
+blinding_private_key := HMAC_SHA256(key=master_blinding_key, msg=script_pubkey)
+blinding_public_key := secp256k1_publickey(private_key=blinding_private_key)
+```
+
+The receiver is using `blinding_public_key` construct a "blinded address", which is used by the sender to blind the relevant transaction outputs. Each such blinded transaction output also contains the sender's ECDH public key, so the receiver would be able to recover the shared nonce using its `blinding_private_key`.
```

Comment:
nits:
 * typo: "to" construct
 * hm, "blinded address" is a somewhat misleading term because the address itself is not blinded

Reply:
Thanks!
Should be fixed at e394776.